src/test.js

```diff
@@ -491,6 +493,13 @@ function synchronize( callback, priority ) {
 
 	if ( priority ) {
 		config.queue.splice( priorityCount++, 0, callback );
+	} else if ( random ) {
+		if ( !unitSampler ) {
+			unitSampler = unitSamplerGenerator( random );
+		}
+
+		index = Math.ceil( unitSampler() * ( config.queue.length - priorityCount ) );
```

Comment:
This expression doesn't look quite right. I think it should be `index = Math.floor( unitSampler() * (config.queue.length - priorityCount + 1) )`, and definitely merits a comment like "Insert into a random queue position after all priority items".